manifests/init.pp

```diff
@@ -1,22 +1,28 @@
-class networkmanager(
-  $version = present,
-  $enable  = true,
-  $start   = true,
-  $gui     = undef,
```

Comment:
Most of these parameters are lost in the refactoring, meaning a breaking change, which could easily be avoided (at least for backward compatibility's sake).